api/openapi/users.yml

```diff
@@ -215,6 +216,42 @@ paths:
           description: Missing or invalid content type.
         '500':
           $ref: "#/components/responses/ServiceError"
+  /users/{userID}/enable:
+    post:
+      summary: Enables a user account
+      description: |
+        Enables a disabled user account given userId.
+      tags:
+        - users
+      responses:       
+        '200':
+          description: User enabled.
+        '400':
+          description: Failed due to malformed JSON.
+        '404':
+          description: Failed due to non existing user.
+        '401':
+          description: Missing or invalid access token provided.
+        '500':
+         $ref: "#/components/responses/ServiceError"
+  /users/{userID}/disable:
+    post:
+      summary: Disables a user account
+      description: |
+        Disables a user account given userId.
+      tags:
+        - users
+      responses:       
```

Comment:
and here. BTW, you can verify with: https://editor.swagger.io/

Reply:
I didn't know about this. Thanks